esd_crawl/settings.py

```diff
@@ -58,12 +58,6 @@
 # See https://docs.scrapy.org/en/latest/topics/extensions.html
 EXTENSIONS = {"scrapy.extensions.closespider.CloseSpider": 500}
 
-# Configure item pipelines
-# See https://docs.scrapy.org/en/latest/topics/item-pipeline.html
-# https://docs.scrapy.org/en/latest/topics/media-pipeline.html#enabling-your-media-pipeline
-ITEM_PIPELINES = {"esd_crawl.pipelines.FindTablePipeline": 300}
-FILES_STORE = "downloads"
```

Comment:
Moved into the ESD crawler since we aren't doing any downloading in the `BrokenSpider`.